tests/test_smoke.py

```diff
@@ -273,7 +273,7 @@ def test_stale_job():
             'sleep 100',  # Ensure this is large enough, else GCP leaks.
             f'sky start {name} -y',
             f'sky logs {name} 1 --status',
-            f's=$(sky queue {name}); printf "$s"; echo; echo; printf "$s" | grep FAILED',
+            f's=$(sky queue {name}); printf "%s\n" "$s"; echo; echo; printf "%s\n" "$s" | grep FAILED',
```

Comment:
Just curious, any reason we use `printf` and manually adding `\n`, instead of using `echo "$s"` directly?